--- browser.js  2020-08-01 10:04:24.442430514 +0900
+++ browser.js  2020-08-01 10:05:15.312026922 +0900
@@ -116,7 +116,7 @@ Browser.create = function create(service

 Browser.defaultResolverSequence = [
   rst.DNSServiceResolve()
-, 'DNSServiceGetAddrInfo' in dns_sd ? rst.DNSServiceGetAddrInfo() : rst.getaddrinfo()
+, 'DNSServiceGetAddrInfo' in dns_sd ? rst.DNSServiceGetAddrInfo() : rst.getaddrinfo({families:[4]})
 , rst.makeAddressesUnique()
 ];

